docs/quickstart.md

```diff
@@ -0,0 +1,129 @@
+---
+post_title: Cassandra Quickstart
+menu_order: 0
+feature_maturity: preview
+enterprise: 'no'
+---
+
+This tutorial will get you up and running in minutes with Cassandra. You will install the DC/OS Cassandra, create a key space, and insert data.
```

Comment:
Suggest: "This tutorial will get you up and running with Cassandra in minutes. You will install the DC/OS Cassandra service..."